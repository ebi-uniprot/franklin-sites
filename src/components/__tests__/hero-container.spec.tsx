import { render } from '@testing-library/react';

import HeroContainer from '../hero-container';

describe('Hero Container component', () => {
  it('should render', () => {
    const { asFragment } = render(
      <HeroContainer title="Title">
        <p>Body content</p>
      </HeroContainer>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
