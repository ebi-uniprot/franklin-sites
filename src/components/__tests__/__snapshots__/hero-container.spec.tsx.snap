// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Container component should render 1`] = `
<DocumentFragment>
  <section
    class="hero-container hero-container--side-padding"
  >
    <h2
      class="hero-container__title big"
    >
      Title
    </h2>
    <p>
      Body content
    </p>
  </section>
</DocumentFragment>
`;
